doc/level1.rst

```diff
@@ -837,8 +837,13 @@ both programmatically and manually.
 - The second mime part of the message MUST have Content-Type
   ``application/autocrypt-setup``, and SHOULD have Content-Disposition
   of ``attachment``. Its content consists of the user's ASCII-armored
-  secret key, encrypted in an ASCII-armored :rfc:`RFC 4880
-  Symmetrically Encrypted Data Packet<4880#section-5.7>`
+  secret key, encrypted with a in an ASCII-armored OpenPGP
```

Comment:
i'm no english expert, but some of "with a in an" looks superfluous to me

Reply:
good catch, @r10s, thanks